targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CYSBSYSKIT_01/CMakeLists.txt

```diff
@@ -60,3 +61,4 @@ target_compile_definitions(mbed-cysbsyskit-01
 )
 
 mbed_post_build_psoc6_merge_hex("CYSBSYSKIT_01")
+mbed_add_component_libraries(mbed-ble mbed-ble-cyw43xxx-43012 mbed-ble-cyw43xxx mbed-ble-cyw43xxx-psoc6)
```

Comment:
Should `mbed-ble-cyw43xxx-43012` and `mbed-ble-cyw43xxx` be here?

Reply:
I'll fix, just one should do the trick if we got deps correct.

Reply:
`mbed-ble-cyw43xxx` is not needed, the first yes - its' firmware. The layout of this is not trivial but seems like both must be defined by a target at the moment.